### ISSUE TITLE
LemonBar crashes on-click

### ISSUE THREAD

**Issue by LordNature** [jaagr/lemonbuddy#34]
In the module for i3 workspaces, if you happen to click in between (or double click on) the workspaces, it happens to crash the bar.

**Log:** 
```
$ lemonbuddy_wrapper top  
terminate called after throwing an instance of 'i3ipc::invalid_reply_payload_error'
  what():  Failed to parse reply on "COMMAND":  first item of root expected to be an object
/usr/bin/lemonbuddy_wrapper: line 41: 12169 Aborted                 (core dumped) $lemonbuddy "$@" -p "$pipe"
```

**Comment by jaagr** [comment]
Hi and thanks for reporting! I'm not using i3 myself so that specific module isn't very well tested. I will fix this issue once I get the time.

**Comment by jaagr** [comment]
Fixed in 2.0, 774dec5

**Comment by BytewaveMLP** [comment]
@jaagr Woot woot!
